src/api/app/controllers/webui/apidocs_controller.rb

```diff
@@ -18,7 +18,7 @@ def file
       send_file( file, :type => "text/xml", :disposition => "inline" )
     else
       flash[:error] = "File not found: #{params[:filename]}"
-      redirect_back_or_to :controller => 'apidocs', :action => 'index'
+      redirect_back(fallback_location: {:controller => 'apidocs', :action => 'index'})
```

Comment:
Yes, but it hurts our eyes!!! :hurtrealbad: 
It won't hurt that much if you "refactor" the same line that you already touched at 50% :smile: